src/package_io/parse_input.star

```diff
@@ -156,6 +156,7 @@ def parse_input(plan, input_args):
 			mev_flood_seconds_per_bundle = result["mev_params"]["mev_flood_seconds_per_bundle"]
 		),
 		launch_additional_services=result["launch_additional_services"],
+		explorer_version=result["explorer_version"],
```

Comment:
this file has changed heavily, since we merged parsing; but these two changes make sense